webpack.config.js

```diff
@@ -170,11 +171,13 @@ async function fetchAppConfigAndEnvironmentVars() {
 
   const { shortlink_domain, thumbnail_server } = hubsConfigs.general;
 
+  const localIp = (await internalIp.v4()) || "localhost";
```

Comment:
The concept of a "local" ip is a bit muddy, you might have multiple "local" interfaces. The lib seems to resolve it based on the default gateway, which is slightly weird conceptually but will probably be the right thing in most cases. All that to say, maybe we want to expose an env var for this to be configured, though I doubt most people would need it.

Reply:
Yup, reused an existing one.